### DISCUSSION TITLE
Uninstall KlipperScreen via terminal

### DISCUSSION THREAD

**Discussion by gixxerfool** [KlipperScreen/KlipperScreen#353] [General]
I cant seem to find instructions for uninstalling KlipperScreen from my Pi.  I started the install and then realized I was not using Moonraker and want to uninstall all the files for KlipperScreen.  Thanks

**Comment by jordanruthe** [comment]
Delete the directory you cloned klipperscreen to and remove /etc/systemd/system/KlipperScreen.service

**Reply by gixxerfool** [reply]
Thanks very much.

**Comment by alfrix** [comment]
also there is an uninstall script in the folder scripts alongside the installer
